pkg/platform/model/vcs.go

```diff
@@ -87,8 +90,8 @@ func NamespaceMatch(query string, namespace NamespaceMatchable) bool {
 type Namespace string
 
 // NamespacePackage creates a new package namespace
-func NamespacePackage(language string) Namespace {
-	return Namespace(fmt.Sprintf("language/%s", language))
+func NamespacePackage(namespace, language string) Namespace {
+	return Namespace(fmt.Sprintf("%s/%s", namespace, language))
```

Comment:
This makes no sense, this function is meant to explicitly construct a package namespace. Making the namespace configurable breaks that contract and renders this function pointless. I think what you want is a second function `NamespaceBundle`. These two concepts are not married.

Reply:
This stemmed from not knowing that the `bundles` namespace does not contain packages whereas the `languages` namespace does.